src/main/java/frc/robot/Match.java

```diff
@@ -125,7 +125,8 @@ public void teleopInit()
   @Override
   public void teleopPeriodic()
   {
-
+    double time = Timer.getMatchTime();
+    if (time < 1 && time>0) robot.intake.retract();
```

Comment:
This retracts at the beginning of the match?

Reply:
The timer counts down to 0 during the match, so it does it at the end.

Reply:
Can you do it in disabledInit()?

Reply:
Maybe, but this works and I don't want to break it for tomorrow

Reply:
Sounds good broski